### PR TITLE
adding python boto3 task

### PR DESCRIPTION
**Pull request by arunvel1988**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Follows the [authoring recommendations][authoring]
- [ ] Includes [docs][docs] (if user facing)
- [ ] Includes [tests][tests] (for new tasks or changed functionality)
  - See the [end-to-end testing documentation][e2e] for guidance and CI details.
- [ ] Meets the [Tekton contributor standards][contributor] (including functionality, content, code)
- [ ] Commit messages follow [commit message best practices][commit]
- [ ] Has a kind label. You can add one by adding a comment on this PR that
  contains `/kind <type>`. Valid types are bug, cleanup, design, documentation,
  feature, flake, misc, question, tep
- [ ] Complies with [Catalog Organization TEP][TEP], see [example]. **Note** [An issue has been filed to automate this validation][validation]
  - [ ] File path follows  `<kind>/<name>/<version>/name.yaml`
  - [ ] Has `README.md` at `<kind>/<name>/<version>/README.md`
  - [ ] Has mandatory `metadata.labels` - `app.kubernetes.io/version` the same as the `<version>` of the resource
  - [ ] Has mandatory `metadata.annotations` `tekton.dev/pipelines.minVersion`
  - [ ] mandatory `spec.description` follows the convention

          ```

          spec:
            description: >-
              one line summary of the resource

              Paragraph(s) to describe the resource.
          ```

_See [the contribution guide](https://github.com/tektoncd/catalog/blob/master/CONTRIBUTING.md) for more details._

[TEP]: https://github.com/tektoncd/community/blob/master/teps/0003-tekton-catalog-organization.md
[example]: https://github.com/tektoncd/catalog/tree/master/task/git-clone/0.1
[validation]: https://github.com/tektoncd/catalog/issues/413
[authoring]: https://github.com/tektoncd/catalog/blob/main/recommendations.md
[docs]: https://github.com/tektoncd/community/blob/master/standards.md#docs
[tests]: https://github.com/tektoncd/community/blob/master/standards.md#tests
[e2e]: https://github.com/tektoncd/catalog/blob/main/CONTRIBUTING.md#end-to-end-testing
[contributor]: https://github.com/tektoncd/community/blob/main/standards.md
[commit]: https://github.com/tektoncd/community/blob/master/standards.md#commit-messages

### CODE REVIEW

**Comment by arunvel1988** [comment]
@vinamra28 do approve its same code base for python boto3 task

File: task/python-boto3-aws/0.1/python-boto3-aws.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      type: string
+      default: "us-east-1"
+      description: AWS Region
+  volumes:
+    - name: python-script
+      configMap:
+        name: python-script-configmap
+  steps:
+    - name: run-python-script
+      image: python:3.9.19-alpine3.20@sha256:45cc18540209d878c2b24080cf8f64fc37603721b67d0ecc508799e2f9a9b21d
```
Comment (vinamra28) [inline task/python-boto3-aws/0.1/python-boto3-aws.yaml:25]:
```suggestion
      image: docker.io/library/python:3.9.19-alpine3.20@sha256:45cc18540209d878c2b24080cf8f64fc37603721b67d0ecc508799e2f9a9b21d
```

File: task/python-boto3-aws/0.1/python-boto3-aws.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: python-boto3
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.18.0"
```
Comment (vinamra28) [inline task/python-boto3-aws/0.1/python-boto3-aws.yaml:8]:
```suggestion
    tekton.dev/pipelines.minVersion: "0.54.0"
```

**Review by vinamra28 — COMMENTED** [review]
hey @arunvel1988, final round of comments and then this can be merged. Please refer to following [link](https://www.git-tower.com/learn/git/faq/git-squash#:~:text=To%20%22squash%22%20in%20Git%20means,often%20done%20when%20merging%20branches.) for squashing the commits.
/ok-to-test

File: task/python-boto3-aws/0.1/python-boto3-aws.yaml (hunk 2 above)
Comment (arunvel1988) [reply task/python-boto3-aws/0.1/python-boto3-aws.yaml]: @vinamra28  done

**Comment by vinamra28** [comment]
@arunvel1988 can you please squash the commits? 🙁 
you can refer the link: https://www.git-tower.com/learn/git/faq/git-squash#:~:text=To%20%22squash%22%20in%20Git%20means,often%20done%20when%20merging%20branches.

**Comment by arunvel1988** [comment]
@vinamra28 done. can u please check

**Comment by vinamra28** [comment]
@arunvel1988 I can still see 2 commits 😅 . Ref: https://github.com/tektoncd/catalog/pull/1279/commits

**Comment by arunvel1988** [comment]
@vinamra28 its done.. i hope

**Comment by vinamra28** [comment]
> @vinamra28 its done.. i hope

@arunvel1988, where are you tracking the commits? 😅 
apologies as our current CI follows `rebase and merge` strategy and we want to maintain clean git history, hence we are suggesting to `squash` the commits. If you check the following link: https://github.com/tektoncd/catalog/pull/1279/commits, it shows two commits.

**Pull request closed without merging**